### DISCUSSION TITLE
Getting query timeout while performing tpch benchmarking on docker 2.0.0_alpha* image.

### DISCUSSION THREAD

**Discussion by abhinav100499** [apache/doris#41178] [A - General / Q&A]
While performing [TPCH Benchmarking](https://doris.apache.org/docs/benchmark/tpch), facing an issue where I had provided 192GB Memory and 48 CPU for each BE container in total I have 3BE and 3FE. Also, exec_mem_limit (GLOBAL) is set to 16GB and cpu is -1 but when I am running benchmarking for 100G i.e. -s 1000, not getting expected response: 
In below table first 2 columns are from official benchmark and last column is from my test result

Query | Apache Doris 2.1.1-rc03 (ms) | Apache Doris 2.0.6 (ms) | Apache Doris   Docker-2.0.0_alpha-*-x86_64
-- | -- | -- | --
Q1 | 7240 | 7638 | Query Timeout
Q2 | 249 | 377 | 53231
Q3 | 3528 | 4389 | 158967
Q4 | 1534 | 1903 | 142141
Q5 | 5457 | 6425 | 94683
Q6 | 159 | 150 | 2310
Q7 | 1598 | 3141 | 105003
Q8 | 2958 | 2792 | 53534
Q9 | 14803 | 24732 | Query Timeout
Q10 | 6743 | 7315 | 140571
Q11 | 414 | 395 | 3888
Q12 | 371 | 417 | 89211
Q13 | 6203 | 8095 | Query Timeout
Q14 | 485 | 681 | 33852
Q15 | 2246 | 1459 | 24051
Q16 | 1252 | 1382 | 27871
Q17 | 1461 | 1204 | Query Timeout
Q18 | 10428 | 11386 | Query Timeout
Q19 | 2329 | 2172 | 11941
Q20 | 569 | 1455 | 84723
Q21 | 5574 | 6570 | Query Timeout
Q22 | 1042 | 1517 | 37038
Total | 76643 | 95595 | 1065392

This is not what expected, and I am not able to debug what properties need to check to get the benchmarking results.

**Comment by ixzc** [comment]
do not use doris-2.0.0_alpha. if you want to test the performance of doris, you need to use the latest and stable version, such as 2.0.14 or 2.1.6

**Reply by abhinav100499** [reply]
Sure, will upgrade the build to 2.1.6, then will publish the stats here.
FYI: following #41173 thread for latest docker image.
